### ISSUE TITLE
Not working on Mac Sequoia

### ISSUE THREAD

**Issue by lazywhite** [zerotier/ZeroTierOne#2380]
Mac: 15.1 Beta (24B5055e)
ZeroTier: 1.14.1

After click ZeroTier, the menu bar icon will appear then disappear, application won't run

**Comment by laduke** [comment]
thanks for reporting. 
If anyone ends up here, try the CLI for now https://docs.zerotier.com/cli/#basic-cli-usage 

`zerotier-cli dump` might be segfaulting too.

**Comment by Teaddy** [comment]
mac: 15.0.1 (24A348)
ZeroTier: 1.14.1

I fixed the problem with the app crashing.  However, there is a new problem,  command `zerotier-cli status` shows `200 info f6c4fxxxxx 1.14.1 ONLINE`, but I can't ping the private IP address in zerotier and I try to connect the device with the private IP address that isn’t working. **Other devices on the Zerotier LAN can ping and access each other.**

**Comment by youde2000** [comment]
搞定了么

**Comment by flrngel** [comment]
@laduke `zerotier-cli` doesn't work after updating mac from 14. My mac is `15.1`, and using`1.14.2` client. It cannot reach to other peers and `zerotier-cli peers` does not show my other machines. All the machines except mac, are working correctly (also using `1.14.2`).

**Comment by laduke** [comment]
I'm not sure. The zerotier-cli command is erroring, or the service isn't able to connect to things?

**Comment by flrngel** [comment]
@laduke If I refresh with `zerotier-cli peers`, it seems like my mac reaches to the machines, but the link first shows RELAY, and then the whole row disappears later. Any guide to debug this issue? I think this is related to privacy & security > local network.

```bash
% sudo zerotier-cli peers
Password:
200 peers
<ztaddr>   <ver>  <role> <lat> <link>   <lastTX> <lastRX> <path>
778cde7190 -      PLANET    94 DIRECT   2679     95412    103.195.103.66/9993
9f77fc393e 1.14.1 LEAF      72 DIRECT   2679     22857    35.208.194.119/32876
cafe04eba9 -      PLANET    -1 RELAY
cafe9ccda7 -      PLANET   181 DIRECT   24760    95327    66.90.98.98/9993
cafe9efeb9 -      PLANET    -1 RELAY

% sudo zerotier-cli info
200 info 079587e78d 1.14.2 ONLINE
```

**Comment by laduke** [comment]
yeah mabny some privacy firewall got turned on in macos. or something unrelated is going on in your router. 

this specific ticket was originally about macOS not letting the program run at all.

**Comment by ethaniel** [comment]
Here is an interesting error:

```
sudo zerotier-cli join *********** (my network id)
objc[12312]: +[NSMutableString initialize] may have been in progress in another thread when fork() was called.
objc[12312]: +[NSMutableString initialize] may have been in progress in another thread when fork() was called. We cannot safely call it or ignore it in the fork() child process. Crashing instead. Set a breakpoint on objc_initializeAfterForkError to debug.
0 join
```

**Comment by laduke** [comment]
that's a new one...
